### PR TITLE
Ondra zizka mavenize migr 233

### PR DESCRIPTION
**Pull request by jsight**
Only relevant commit: https://github.com/OndraZizka/windup/commit/aa2772fd0b06d1ac58458decdf708bf0501071fe

### CODE REVIEW

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ModuleAnalysisHelper.java [hunk 1]
```diff
@@ -26,36 +17,45 @@
...
-
-    GraphContext grCtx;
-
-
-    public ModuleAnalysisHelper(GraphContext grCtx)
+    public ModuleAnalysisHelper(GraphContext context)
     {
-        this.grCtx = grCtx;
+        this.context = context;
     }
```
Comment (OndraZizka) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ModuleAnalysisHelper.java]: I don't think this improves readability.

File: config/api/src/main/java/org/jboss/windup/config/query/QueryPropertyCriterion.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ public void query(GraphRewrite event, GremlinPipeline<Vertex, Vertex> pipeline)
         }
     }
 
-    public final static class MultipleValueTitanPredicate implements TitanPredicate
+    private final static class MultipleValueTitanPredicate implements TitanPredicate
     {
```
Comment (OndraZizka) [inline config/api/src/main/java/org/jboss/windup/config/query/QueryPropertyCriterion.java]: This isn't reusable?

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ArchiveCoordinateService.java [hunk 3]
```diff
@@ -1,16 +1,17 @@
 package org.jboss.windup.rules.apps.mavenize;
 
-
 import java.util.Iterator;
 import java.util.logging.Logger;
+
```
Comment (OndraZizka) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ArchiveCoordinateService.java]: Maybe we should remove imports formatting from the formatting profile...

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/MavenizationService.java [hunk 4]
```diff
@@ -205,7 +205,7 @@ else if (true /* isLibraryAndNotProjectModule() */)
             // Known library -> simple dependency.
             if(file instanceof IdentifiedArchiveModel){
                 IdentifiedArchiveModel artifact = (IdentifiedArchiveModel) file;
-                modulePom.dependencies.add(new SimpleDependency(Dependency.Role.LIBRARY, new MavenCoord(artifact.getCoordinate())));
+                modulePom.dependencies.add(new SimpleDependency(Dependency.Role.LIBRARY, MavenCoord.from(artifact.getCoordinate())));
             }
```
Comment (OndraZizka) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/MavenizationService.java]: Thanks :)

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ModuleAnalysisHelper.java (hunk 1 above)
Comment (jsight) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ModuleAnalysisHelper.java]: Yeah, graphContext is preferred. We are generally avoiding abbreviations.

File: config/api/src/main/java/org/jboss/windup/config/query/QueryPropertyCriterion.java (hunk 2 above)
Comment (jsight) [inline config/api/src/main/java/org/jboss/windup/config/query/QueryPropertyCriterion.java]: At the moment it is not.

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ArchiveCoordinateService.java (hunk 3 above)
Comment (jsight) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ArchiveCoordinateService.java]: I wish that we were consistent in formatting them. I don't see it as a big issue either way, though. The auto-formatters seem to behave somewhat inconsistently.

**Pull request closed without merging**